jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -32,6 +32,48 @@
         - 'test-infra/.git/'
         external-deletion-command: 'sudo rm -rf %s'
 
+- job-template:
+    builders:
+    - shell: |
+        # TODO(fejta): consider a stable tag instead of master
+        git clone https://github.com/kubernetes/test-infra -b master
+        './test-infra/jenkins/bootstrap.py' --job='{job-name}' --bare --root="${{GOPATH}}/src/k8s.io/kubernetes"
```

Comment:
what does `--root` mean in the context of a `--bare` job? we aren't checking anything out here, so creating a git-like gopath seems odd.